core/chains/evm/headtracker/head_broadcaster_test.go

```diff
@@ -19,13 +19,14 @@ import (
 	commonhtrk "github.com/smartcontractkit/chainlink-framework/chains/headtracker"
 
 	commonmocks "github.com/smartcontractkit/chainlink/v2/common/types/mocks"
-	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/config/toml"
 	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/headtracker"
 	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/headtracker/mocks"
 	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/headtracker/types"
-	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/testutils"
-	evmtypes "github.com/smartcontractkit/chainlink/v2/core/chains/evm/types"
 	"github.com/smartcontractkit/chainlink/v2/core/internal/testutils/pgtest"
+	"github.com/smartcontractkit/chainlink/v2/evm/client/clienttest"
+	"github.com/smartcontractkit/chainlink/v2/evm/config/toml"
+	testutils2 "github.com/smartcontractkit/chainlink/v2/evm/testutils"
```

Comment:
Curious why renaming testutils to testutils2 here

Reply:
Ah that was just goland being foolish. I thought I had cleaned them all up. I'll follow-up to fix these.